src/diff/children.js

```diff
@@ -142,6 +143,7 @@ export function diffChildren(
 		} else {
 			// this was previously captured after patch(), which seems backwards?
 			oldVNodeRef = childInternal.ref;
+			prevDom = childInternal._dom;
```

Comment:
Argh, really wondering if there is a better way to do this... It's required for a test or two in `placeholders.test.js` I believe

Reply:
Added an inline TODO and a follow up to start at this code a bit more to find a more elegant way to do this.